benches/decode.rs

```diff
@@ -36,7 +36,7 @@ fn decode_stream(c: &mut Criterion) {
             b.iter(|| {
                 let i = input.clone();
                 let mut input_r = std::io::Cursor::new(i);
-                let options = yenc::DecodeOptions::new("/tmp");
```

Comment:
This change is not backward compatible, breaking existing clients.
If adding new options, add a new function, like done in  EncodeOptions.